tests/apps/test_similarity.py

```diff
@@ -340,6 +352,26 @@ def test_odd_photon_numbers(self, k, nmax):
         graph = nx.complete_graph(10)
         assert similarity.prob_event_exact(graph, k, nmax) == 0.0
 
+    def test_correct_result_returned(self, monkeypatch):
+        """Tests if the call to _get_state function is performed correctly and probabilities over all
+        constituent orbits of the given event are summed correctly. The test monkeypatches the fock_prob
+        function so that the probability is the same for each sample permutation of all constituent orbits
+        and is equal to 1/8. For a 4-mode graph, an event with ``photon_number = 2``, and
+        ``max_count_per_mode = 1`` contains orbit [1, 1] which has 6 possible sample permutations."""
+        graph = nx.complete_graph(4)
+        with monkeypatch.context() as m:
+            m.setattr("strawberryfields.backends.gaussianbackend.states.GaussianState.fock_prob",
+                      lambda *args, **kwargs: 1/8)
+            assert similarity.prob_event_exact(graph, 2, 1) == 6/8
+
+    def test_known_result(self):
```

Comment:
Same comment as with the test above: better to test that we reproduce a hard-coded value when the mean photon number is larger